custom/build.gradle.kts

```diff
@@ -28,20 +28,29 @@ dependencies {
   testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
   testImplementation("io.micrometer:micrometer-core")
 
-  testImplementation(project(':testing:common'))
-  testImplementation 'javax.servlet:javax.servlet-api:3.0.1'
-  testImplementation 'org.eclipse.jetty:jetty-server:8.0.0.v20110901'
-  testImplementation 'org.eclipse.jetty:jetty-servlet:8.0.0.v20110901'
+  testImplementation(project(":testing:common"))
+  testImplementation("javax.servlet:javax.servlet-api:3.0.1")
+  testImplementation("org.eclipse.jetty:jetty-server:8.0.0.v20110901")
+  testImplementation("org.eclipse.jetty:jetty-servlet:8.0.0.v20110901")
+}
+
+sourceSets {
+  main {
+    output.dir("build/generated/properties", "builtBy" to "generateVersionResource")
+  }
 }
 
 tasks {
   compileJava {
     options.release.set(8)
   }
 
-  processResources {
-    expand([
-        'version': project.version
-    ])
+  register("generateVersionResource") {
```

Comment:
Why this change? Kotlin script does not support `processResources` task?

Reply:
Honestly, I don't know - it passed locally on my machine, it failed in GHA:

```
Execution failed for task ':custom:processResources'.
> Could not copy file '/home/runner/work/splunk-otel-java/splunk-otel-java/custom/src/main/resources/splunk.properties' to '/home/runner/work/splunk-otel-java/splunk-otel-java/custom/build/resources/main/splunk.properties'.
   > java.lang.UnsupportedOperationException (no error message)
```

I couldn't reproduce it so I just looked up how OTel SDK does it and copied the solution.

Reply:
Can you add some TODO here? This is certainly a hack